datasets/gov/lsd/cl/ja_asmenys.csv

```diff
@@ -7,23 +7,23 @@ id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri
 ,,,,,,comment,type,,"update(type: ""ref"", ref: ""/datasets/gov/lsd/cl/evrk/EkonominesVeiklosRusis"")",4,open,spinta:1476,,
-,,,,,ja_veikla_atnaujinta,datetime,,,,2,open,,"Pagrindinės veiklos rūšies atnaujinimo data (Veiklos atnaujinimo data)","Data, kada paskutinį kartą buvo atnaujintas juridinio asmens pagrindinės veiklos rūšies kodas."
+,,,,,veiklos_data,datetime,,,,2,open,,"Pagrindinės veiklos rūšies atnaujinimo data (Veiklos atnaujinimo data)","Data, kada paskutinį kartą buvo atnaujintas juridinio asmens pagrindinės veiklos rūšies kodas."
 ,,,,,,comment,"property, type",,"update(property: ""veikla_atnaujinta"")",4,open,vadovas:dsa/kodiniai-pavadinimai.html,,
 ,,,,,sektorius,string,,,,2,open,,"Institucinis sektorius ir subsektorius (Institucinis sektorius)","Ūkio subjektai (fiziniai ir juridiniai asmenys), kurie gali savarankiškai nuosavybės teise turėti prekių ir turto, prisiimti įsipareigojimus, vykdyti ekonominę veiklą ir sudaryti sandorius su kitais vienetais vadinami instituciniais vienetais. ESS sistemoje instituciniai vienetai skirstomi į penkis tarpusavyje nesidubliuojančius šalies institucinius sektorius. Šie penki sektoriai sudaro visą šalies ekonomiką.  Vienetai nerezidentai gali būti susiję su visais šiais penkiais šalies sektoriais ir šios sąsajos pateikiamos kaip šalies sektorių ir šeštojo sektoriaus – likusio pasaulio sektoriaus – ryšiai. Sektoriai skirstomi į subsektorius. Ūkio subjektų instituciniai sektoriai ir subsektoriai nustatomi, tikslinami ir koduojami statistiniame ūkio subjektų registre nuolat remiantis administraciniais ir statistiniais duomenų šaltiniais. Šie duomenysyra informacijos šaltinis valstybės institucijoms, privačioms bendrovėms ir organizacijoms. Jie yra naudojami sudarant nacionalines sąskaitas, mokėjimų balansą, pinigų ir finansų statistikai rengti."
 ,,,,,,comment,type,,"update(type: ""ref"", ref: ""datasets/gov/lsd/cl/sektoriai/Sektorius"")",4,open,spinta:1476,,
-,,,,,reg_data,date,D,,,4,open,,"Įregistravimo data","Juridinio asmens įregistravimo data (JAR duomuo)"
+,,,,,iregistruotas,date,D,,,4,open,,"Įregistravimo data","Juridinio asmens įregistravimo data (JAR duomuo)"
 ,,,,,nforma,integer,,,,4,open,,"Nuosavybės formos","Nuosavybės formos"
```

Comment:
```suggestion
,,,,,nforma,integer,,,,2,open,,"Nuosavybės formos","Nuosavybės formos"
```
Jeigu šis duomuo yra iš JAR, tuomet brandos lygis turėtų būti 2, nes nepadaryta sąsaja su JAR.

Reply:
jeigu duomenys "ateine" iš JAR ir nepadarytas susiejimas, tai brandos lygis yra 2, net jeigy yra nurodytas `enum`.

Reply:
Šitas rinkinys yra pirminis , sukurtas LSD, ar tikrai palikti lygį=2 ?